src/Processors/Transforms/Streaming/AggregatingTransformWithSubstream.cpp

```diff
@@ -294,7 +294,7 @@ void SubstreamContext::serialize(WriteBuffer & wb, VersionType version) const
 {
     DB::Streaming::serialize(id, wb);
 
-    aggregating_transform->params->aggregator.checkpoint(variants, wb);
+    DB::serialize(variants, wb, aggregating_transform->params->aggregator);
```

Comment:
i personally feel like the previous version / style might be better than free function. 

```
variants.serialize(wb, aggregator);
```